addons/account/models/chart_template.py

```diff
@@ -243,6 +244,18 @@ def load_for_current_company(self, sale_tax_rate, purchase_tax_rate):
         # Install all the templates objects and generate the real objects
         acc_template_ref, taxes_ref = self._install_template(company, code_digits=self.code_digits)
 
+        # Set default PoS receivable account in company
+        if acc_template_ref.get(self.default_pos_receivable_account_id.id):
+            company.write({
+                'default_pos_receivable_account_id': acc_template_ref[self.default_pos_receivable_account_id.id]
```

Comment:
remove 'default_pos_receivable_account_id' from company, and use property, if no field in the localisation, no default value needed.

Reply:
LAP wanted bank and cash payment methods to be created when account.chart.template is installed. pos.payment.method requires receivable_account_id (for both payment methods) and cash_journal_id (for the cash payment method) -- this is not negotiable as the spec is dependent on these fields, specially on receivable_account_id field. Moreover, it is explicit that a receivable account different from the default receivable account should be created. Without the right name of this receivable account, I think copying the property_account_receivable_id and using the name with 'PoS' [maybe _('Point of Sale') is better] at the end is really a good alternative, than not creating an account.

Reply:
As we don't use property anymore, you can forget comments in the review concerning property (like this one)